### PR TITLE
Add Field tooltip

### PR DESCRIPTION
**Pull request by arkadiuszbachorski**
# Add Field tooltip

## :recycle: Current situation & Problem
Fields sometimes need more explanation about their purpose or details. Tooltips are useful for that. 

## :gear: Release Notes 
* Add Field tooltip

## :white_check_mark: Testing
image

## :pencil: Code of Conduct & Contributing Guidelines 

By submitting creating this pull request, you agree to follow our [Code of Conduct](https://github.com/StanfordSpezi/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/StanfordSpezi/.github/blob/main/CONTRIBUTING.md):
- [x] I agree to follow the [Code of Conduct](https://github.com/StanfordSpezi/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/StanfordSpezi/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

**Review by ThomasKaar — APPROVED** [review]
Thanks a lot for adding these components!!

Looks great!

I just added some minor comments

File: src/forms/Field/FieldTooltip.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import { Tooltip } from '@/components/Tooltip'
+import { type FieldProps } from '@/forms'
+import { ensureString } from '@/utils/misc'
+
+interface FieldTooltipProps extends Pick<FieldProps, 'tooltip' | 'label'> {
+  id: string
+}
+
+export const FieldTooltip = ({ tooltip, label, id }: FieldTooltipProps) => (
+  <Tooltip tooltip={tooltip} open={true}>
```
[the remark this answers is not in the document]
Comment (arkadiuszbachorski) [reply src/forms/Field/FieldTooltip.tsx:19]: Good catch, I just did it for testing purposes and committed accidentally!

[the remark this answers is not in the document]
Comment (arkadiuszbachorski) [reply src/forms/Field/FieldTooltip.tsx]: [Spread FieldTooltip props](https://github.com/StanfordSpezi/spezi-web-design-system/pull/6/commits/4b83e9fee6f1e22529ce827e75bc6e85d681c50b)

File: src/forms/Field/Field.test.tsx [hunk 2]
```diff
@@ -67,4 +67,17 @@ describe('Field', () => {
...
+
+  it('renders accessible tooltip', async () => {
+    const user = userEvent.setup()
+    render(<Component tooltip="Tooltip content" />)
+
+    const tooltipTrigger = screen.getByLabelText(
+      'More information about the name field',
+    )
+
+    await user.hover(tooltipTrigger)
```
[the remark this answers is not in the document]
Comment (arkadiuszbachorski) [reply src/forms/Field/Field.test.tsx:80]: [Add additional test expectation before interaction](https://github.com/StanfordSpezi/spezi-web-design-system/pull/6/commits/78178678f215e8c77f5fc6e144de21288e243b5c)

**Review by NikolaiMadlener — COMMENTED** [review]
Nice addition @arkadiuszbachorski ! 🚀

**Review by NikolaiMadlener — APPROVED** [review]
Nice addition @arkadiuszbachorski ! 🚀

**Pull request merged**